documentation/docs/payid.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: payid
+title: PayID
+---
+
+import { Mermaid } from './Mermaid'
+
+By default, Open Payments interactions centre around an [account](./accounts)
+resource and all APIs are relative to the account URL.
+
+The account URL is [discovered](./discovery) through a
+[Payment Pointer](./terminology#payment-pointer) which is an account identifer
+for Interledger-enabled accounts.
+
+Said differently, any account identified by a Payment Pointer can be paid into
+using the [STREAM](terminology#STREAM) protocol.
+
+This feature allows clients to setup payments, for uses cases such
+[web monetization](./web-monetization), with a single API call, to get the
+payment details, and be guaranteed that the receiving account supports ILP as
+payment method.
+
+A **PayID** is an alternative identifier which, unlike Payment Pointers that
+identify accounts, identifies [account owners](./terminology#account-owner).
+
+The
+[PayID URI specification](https://github.com/xpring-eng/rfcs/blob/master/payid/dist/spec/payid-uri.txt)
+defines the format of a PayID to be `acctpart "$" host`.
+
+With respect to Open Payments the `acctpart` is the account owner identifier and
+`host` is the origin of the Open Payments server.
+
+The following PayID identifies an account for the owner `alice` at the wallet
+`wallet.example`:
+
+> `alice$wallet.example`
+
+## Open Payments PayID Discovery
+
+As with all APIs in Open Payments the [owner](./owner.mdx) is a resource and is
+identified by a URL.
+
+The
+[PayID URI specification](https://github.com/xpring-eng/rfcs/blob/master/payid/dist/spec/payid-uri.txt)
+allows for any protocol to define a discovery protocol on top of the identifier.
+For Open Payments the discovery protocol is as follows:
+
+1. Given a PayID conforming to the prescribed format, `acctpart "$" host`,
+   extract the values **acctpart** and **host**.
+2. Let **path** be the string `/$/` concatenated with **acctpart**
+3. Let **account owner URL** be a new URL as defined in
+   [WHATWG URL](https://url.spec.whatwg.org/#concept-url) where:
+   1. The URL's scheme is `https`
+   2. The URL's host is **host**
+   3. The URL's path is **path**
+
+For example, the PayID `alice$wallet.example` resolves to the URL
+`https://wallet.example/$/alice`.
+
+The `/$/` path prefix minimizes the chance that Web servers will have path
+conflicts with their existing services.
+
+## Privacy
```

Comment:
Relating to PayID over OpenPayments, one thing my team has been exploring a lot is where it makes sense to get certain data in non-ILP open-payments flows (e.g.,  PayID + XRPL). E.g., sometimes it makes sense to put data into the OP layer, and sometimes it makes sense to defer to the PayID layer.

Probably something we should add to the agenda for Wednesday's call -- but in short I think in OP primitives, we're missing the concept of a "Payment Network" when it comes to something like an `invoice` or a `mandate`. 

For example, we have an `address` (PaymentPointer or XRPL addr or BTC addr or whatever); we have the `assetCode` and `assetScale`, but we're missing the underlying network.

As currently written, the OP spec assume ILP only, so the payment network is inferred throughout the protocol.

Reply:
> I think in OP primitives, we're missing the concept of a "Payment Network" when it comes to something like an invoice or a mandate.

I don't agree. The payment network is implied by the supported media types. As a client you can request an invoice or account resource and specify all the ways you are able to pay into that resource (i.e. all the networks you are on) and the response provides the payment details for that network embedded in the resource.

> For example, we have an address (PaymentPointer or XRPL addr or BTC addr or whatever); we have the assetCode and assetScale, but we're missing the underlying network.

A Payment Pointer is not an address, it's an identifier, like a PayID. The address is an ILP address.

> As currently written, the OP spec assume ILP only, so the payment network is inferred throughout the protocol.

Not true. OP mandates ILP as a supported network if the discovery was done from a Payment Pointer. This ensures backward compatibility with the existing ecosystem where Payment Pointers are always identifying ILP-enabled wallets.

If you have an account or invoice resource URL and you query it with other accepted media types you can get a payment address for other networks.